examples/llm/README.md

```diff
@@ -58,6 +58,14 @@ pip install -e ".[llm]"  # or pip install -e ".[llm-cpu]" if no NVIDIA GPU
 cd examples/llm
 ```
 
+If you have an NVIDIA GPU, you can install our library of CUDA optimizations to speed up your models.
+```bash
+cd csrc
+pip install .  # may take a long time (up to 20 minutes)
```

Comment:
can you take a todo to decide whether we should ship a docker image with all of this preinstalled?